LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 Vesoft, Inc.
```

Comment:
why change

Reply:
This is supposed to be filled. Reference: https://github.com/docker/compose/blob/master/LICENSE